nautobot/extras/views.py

```diff
@@ -187,6 +188,66 @@ class ConfigContextUIViewSet(NautobotUIViewSet):
     serializer_class = serializers.ConfigContextSerializer
     table_class = tables.ConfigContextTable
 
+    class AssignmentObjectFieldsPanel(object_detail.ObjectFieldsPanel):
+        def render_value(self, key, value, context):
+            if not value or (key == "dynamic_groups" and not settings.CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED):
+                return helpers.HTML_NONE
+
+            items = []
+            for val in value.all():
+                if isinstance(val, Role):
```

Comment:
Can you explain this? I don't see the same logic in the template so trying to understand where this comes from

Reply:
Let's remove the special-case behavior for `Role` and let it be treated like all the others. I suspect the current special-case is a carryover from 1.x when `DeviceRole` was its own model.